log/logger_setup.go

```diff
@@ -100,10 +100,18 @@ func SetupSubLoggers(s []SubLoggerConfig) error {
 }
 
 // SetupGlobalLogger setup the global loggers with the default global config values
-func SetupGlobalLogger() error {
+func SetupGlobalLogger(workerCount int) error {
 	RWM.Lock()
 	defer RWM.Unlock()
 
+	close(workerShutdown)
+	workerWg.Wait()
+	workerShutdown = make(chan struct{})
+	for x := 0; x < workerCount; x++ {
+		workerWg.Add(1)
+		go loggerWorker()
+	}
```

Comment:
```suggestion
if workerCount > 1 {
	close(workerShutdown)
	workerWg.Wait()
	workerShutdown = make(chan struct{})
	for x := 0; x < workerCount; x++ {
		workerWg.Add(1)
		go loggerWorker()
	}
}
```
No need to do all that if someone sets it to 1 for whatever reason